docs/src/components/ServiceUnitTest.tsx

```diff
@@ -0,0 +1,65 @@
+/// <reference path="../../../jacdac-spec/spectool/jdtest.d.ts" />
+
+import React, { useContext, useState } from 'react';
+import Markdown from "./ui/Markdown"
+import { Link } from 'gatsby-theme-material-ui';
+import useGridBreakpoints from './useGridBreakpoints';
+
+import useChange from '../jacdac/useChange';
+import { Grid, Card, CardHeader, CardActions, Button, createStyles, makeStyles, Paper, Step, StepContent, StepLabel, Stepper, Theme, Typography } from '@material-ui/core';
+// tslint:disable-next-line: no-submodule-imports
+import Alert from "./ui/Alert";
+import DeviceCardHeader from "./DeviceCardHeader"
+import { JDService } from '../../../src/jdom/service';
+
+enum TestStatus {
+    Inactive,
+    Active,
+    Passed,
+    Failed,
+}
+
+export function ServiceUnitTest(props: {serviceInstance: JDService, test: jdtest.UnitTest}) {
+    const { serviceInstance, test } = props
+    const [ testStatus, setTestStatus ] = useState<TestStatus>(TestStatus.Inactive)
+    return (
+        <Step key={test.description}>
+        <StepLabel>{test.description}</StepLabel>
```

Comment:
do you expect ``test.description`` to be markdown? you can use ``<Markdown ... />``